src/services/vision.ts

```diff
@@ -23,14 +23,14 @@ export function convertCompositeApyToSnakeCase(apy: Apy | undefined): Apy | unde
               boost: apy.composite.boost
                 ? apy.composite.boost
                 : (apy.composite as unknown as BackscracherApyComposite).currentBoost,
-              pool_apy: apy.composite.pool_apy
-                ? apy.composite.pool_apy
+              poolApy: apy.composite.poolApy
+                ? apy.composite.poolApy
                 : (apy.composite as unknown as BackscracherApyComposite).poolApy,
-              boosted_apr: apy.composite.boosted_apr
-                ? apy.composite.boosted_apr
+              boostedApr: apy.composite.boostedApr
```

Comment:
We cannot do this because that's the key that comes from the API, see https://api.yearn.finance/v1/chains/1/vaults/all